playbooks/openshift/roles/setup/handlers/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: Restart Docker Service
+  service: name=docker state=restarted
```

Comment:
nit:  I need to go back through our OSE ones but we should be consistent and use the:
name: docker
state: restarted

style